srv/modules/runners/upgrade.py

```diff
@@ -106,6 +108,32 @@ def deprecated_straw(self):
         return True, ""
 
 
+class OrigDriveGroup:
```

Comment:
We could feed this thing a couple of known drivegroups in and validate the outcome.

That should be really straight forward as we're dealing only static data here.